plugins/scaffolder-backend/src/scaffolder/tasks/TaskWorker.ts

```diff
@@ -156,8 +156,15 @@ export class TaskWorker {
     })();
   }
 
-  stop() {
+  stop(options?: { waitForTaskCompletion?: boolean }) {
```

Comment:
I wonder if we should just have this as part of the `CreateWorkerOptions` and pass through to the constructor instead?

Can be optional in that instead?

Reply:
true, will change!